src/bokeh/models/glyphs.py

```diff
@@ -1208,7 +1208,13 @@ def __init__(self, *args, **kwargs) -> None:
     """)
 
 class Rect(XYGlyph, LineGlyph, FillGlyph, HatchGlyph):
-    ''' Render rectangles.
+    ''' Render rectangles, characterised by x and y position, width, height,
```

Comment:
`x` and `y` are coordinates of the center of the glyphs, which I think should be mentioned here.

Reply:
OK.

Reply:
Although on log axes it can only be the center in either data coordinates or screen coordinates, not both. But let's pretend that I haven't said that!